cli/cmd/compose/compose.go

```diff
@@ -103,18 +103,26 @@ func (o *projectOptions) toProjectOptions() (*cli.ProjectOptions, error) {
 func Command(contextType string) *cobra.Command {
 	opts := projectOptions{}
 	var ansi string
+	var noAnsi bool
 	command := &cobra.Command{
 		Short:            "Docker Compose",
 		Use:              "compose",
 		TraverseChildren: true,
 		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			if noAnsi {
+				if ansi != "auto" {
+					return errors.New(aec.Apply(`cannot specify DEPRECATED "--no-ansi" and "--ansi". Please use only "--ansi".`, aec.RedF))
```

Comment:
is it relevant to apply colors to golang error message?
I'm fine we do when we directly display message on stderr, but errors we forward to caller should be plain text

Reply:
no indeed, an error in CC/paste